### DISCUSSION TITLE
Who would you like to see as a speaker for BEJUG?

### DISCUSSION THREAD

**Discussion by TomCools** [BeJUG/bejug.github.io#20] [Ideas]
Comment with a list of people you'd like to see in a future event? It can literally be anyone!  We can see how we can accommodate this!

**Comment by SimonVerhoeven** [comment]
Oh boy, it's far from an exhaustive list, but:

Maciej Walkowiak
DaShaun Carter
Dan Vega
Holly Cummins
Josh Long
Mala Gupta
Rafael del Nero
Trisha Gee
Ken Kousen
Felienne Hermans
Sharat Chander
Tim Te Beek
Julien Dubois
Oleg Nenashev
Oleg Šelajev
Arun Gupta
Thomas Vitale
Nicolai Parlog
Jose Paumard
Tagir Valeev
Venkat Subramaniam
Kevlin Henney
Miroslav Wengner
Thomas Schühly
Laurentiu Spilca
Brian Vermeer
Lukas Eder
Simone De Gijt
Philip Riecks
Mark Heckler
Ana Maria Mihalceanu
Alina Yurenko
Helen Scott
Marit Van Dijk
Simon Martinelli
Karl Heinz Marbaise
Roy Van Rijn
Simon Maple
Marcus Hellberg
Vlad Mihalcea
...

**Comment by SimonVerhoeven** [comment]
Lars Rosenquist mentioned he might be interested in giving a talk (mayhaps with Dieter Hubau)
